src/main/clm_instMod.F90

```diff
@@ -214,6 +220,9 @@ subroutine clm_instInit(bounds)
     begc = bounds%begc; endc = bounds%endc 
     begl = bounds%begl; endl = bounds%endl
 
+    call getfil (paramfile, locfn, 0)
+    call ncd_pio_openfile (params_ncid, trim(locfn), 0)
```

Comment:
Note that I open the param file here then pass the ncid into the init function below, rather than opening the file there. There were two reasons for doing it this way:

1. I envision needing to have other parameterizations read from the param file in their init methods, and I was concerned about the performance implications of repeatedly opening and closing the file.

2. This would lend itself better to unit testing, if we ever want to pass in a fake ncid for use in unit tests.